Setup.bat

```diff
@@ -122,6 +122,7 @@ call :MarkStartOfBlock "Retrieve dependencies"
     call :ExecuteAndCheck spatial package retrieve worker_sdk    c_headers                                  %PINNED_CORE_SDK_VERSION%    %DOMAIN_ENVIRONMENT_VAR%    "%CORE_SDK_DIR%\worker_sdk\c_headers.zip"
     call :ExecuteAndCheck spatial package retrieve worker_sdk    c-dynamic-x86_64-vc141_md-win32            %PINNED_CORE_SDK_VERSION%    %DOMAIN_ENVIRONMENT_VAR%    "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-vc141_md-win32.zip"
     call :ExecuteAndCheck spatial package retrieve worker_sdk    c-dynamic-x86_64-clang1000-linux           %PINNED_CORE_SDK_VERSION%    %DOMAIN_ENVIRONMENT_VAR%    "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-clang1000-linux.zip"
+    call :ExecuteAndCheck spatial package retrieve internal      etlog-x86_64-win32                         %PINNED_CORE_SDK_VERSION%    %DOMAIN_ENVIRONMENT_VAR%    "%CORE_SDK_DIR%\tools\etlog-win32.zip"

```

Comment:
not sure what happens when someone without permissions tries to download something that's `internal`, can you comment @joshuahuburn ?

Reply:
hm also, I didn't include the linux etlog package in Setup.sh, will we need that?

Reply:
Hmmm, most likely we will want this behind a flag wall if it requires special perms

Reply:
right, gated it with --internal now `./Setup.bat --internal`

Reply:
what about the linux distro?

Reply:
I think the "internal" package type is just a convention. The packages appear to be "unrestricted":
```
$ package_client list internal etlog-x86_64-linux 15.2.0
+000 Retrieved information for package type 'internal' with name 'etlog-x86_64-linux' and version '15.2.0'
+000 {
     	type: internal
     	name: etlog-x86_64-linux
     	version: 15.2.0
     	sdk min version: 
     	restriction: unrestricted
     	creation time: 2021-07-15 13:34:52.863985 +0000 UTC
     	creator email: 
     	md5 checksum: oiaDnsQ6H9Mx6RmrdPMg1w==
     	download URL: https://storage.googleapis.com/improbable-packages-private/internal/etlog-x86_64-linux/15.2.0/a226839ec43a1fd331e919ab74f320d7?Expires=1626959467&GoogleAccessId=56260027111-427eh55vcs7gne2i6d6qaa60egsj9k6t%40developer.gserviceaccount.com&Signature=eoG70N79Nen16%2FlTYRYjWguiDNbl5UzXOshH4Hkw5QqwrU8htm1uuQdKCfB%2Bu3aLtKR72KwPAKz3%2FkFsP4HrIXgFmyw3OTL2NjEXu1EIf509xnY9KfW%2Fzg6FvYBsb9DpWSavgkoAfui6LCXoacRlVVIpZPUiFId7O6N%2BnwzQaZo%3D
     }
```

Reply:
aight, I guess we can remove the flag if/when it becomes more public